core-bundle/contao/themes/flexible/basic.css

```
@@ -77,61 +77,67 @@
 	--nested-bg: #fbfbfd;
 }
 
-@media (prefers-color-scheme: dark) {
-	:root {
-		--text: #ddd;
-		--body-bg: #121416;
-		--content-bg: #1b1d21;
-		--content-border: #4e5156;
-		--black: #fff;
-		--white: #000;
-		--blue: #0073a8;
-		--contao: #f47c00;
-		--border: #3a3c40;
-		--nav-bg: #1b1d21;
-		--nav-hover-bg: #1b325f;
-		--nav-current: #292c32;
-		--nav-separator: #3f3f3f;
-		--hover-row: #1b325f;
-		--header-bg: #292c32;
-		--header-bg-hover: #202327;
-		--invert-bg: #7e8695;
-		--invert-text: #222;
-		--table-header: #232529;
-		--table-odd: #1b1d21;
-		--table-even: #1e2024;
-		--table-nb-header: #292c32;
-		--table-nb-odd: #1b1d21;
-		--table-nb-even: #23252a;
-		--panel-bg: #292c32;
-		--tree-header: #292c32;
-		--tree-header-border: #3f4146;
-		--form-text-disabled: #666;
-		--form-border: #44464b;
-		--form-border-disabled: #3a3c40;
-		--form-bg: #151619;
-		--form-bg-hover: #1e2024;
-		--form-bg-disabled: #1e2024;
-		--form-button: #31333a;
-		--form-button-hover: #383a42;
-		--form-button-active: #777;
-		--form-button-disabled: #26272c;
-		--diff-left: rgba(248,81,73,0.15);
-		--diff-del: rgba(248,81,73,0.4);
-		--diff-right: rgba(46,160,67,0.15);
-		--diff-ins: rgba(46,160,67,0.4);
-		--code-bg: #30343b;
-		--checkerbox-bg: #30343b;
-		--info: #9095a2;
-		--active-bg: #1b325f;
-		--active-border: #264787;
-		--drag-bg: #1b325f;
-		--legend: #747b8b;
-		--serp-preview: #bdc1c6;
-		--serp-preview-title: #8ab4f8;
-		--nested-bg: #1e2024;
-		--datepicker-hover: #393e46;
-	}
+html[data-color-scheme="dark"] {
```

```suggestion
html[data-color-scheme="dark"] {
	color-scheme: dark;
```

This is needed so that the UI elements of the browser (e.g. checkboxes) also switch to dark mode.

---

added in e5ceb2fb7689609127c7e7f38d549432b2bca5ff